code/_onclick/item_attack.dm

```diff
@@ -7,6 +24,10 @@
 
 
 
+/obj/item/proc/pre_attack(atom/A, mob/living/user, params) //do stuff before attackby!
```

Comment:
This proc is not really necessary. Remove it, same as the check, until it's actually needed/useful.

Reply:
Shall I comment it out for now then?

Reply:
Remove instead.